README

```diff
@@ -6,54 +6,67 @@ plugins to build large scale web applications.
-      |-- config/ ........ Configurations
-      |-- hdrs/ .......... Headers parsing and management
-      |-- http/ .......... The actual HTTP protocol implementation
-      |---http2/ ......... HTTP/2 implementation
-      |-- logging/ ....... Flexible logging
-  |-- README ............. intro, links, build info
-  |-- README-EC2 ......... info on EC2 support
-  |-- REVIEWERS .......... (incomplete) list of areas with committer interest
-  |-- LICENSE ............ full license text
-  `-- NOTICE ............. copyright notices
+  trafficserver/ ............ Top src dir
+  |-- build/ ................ Custom macros for configure.ac
+  |-- ci/ ................... Quality assurance and other CI tools and configs
+  |-- contrib/ .............. Various contributed auxiliary pieces
+  |-- doc/ ..................
+      |-- admin/ ............ Admin guide documentations
+      |-- dot/ .............. Graphviz source files for docs pictures
+      |-- man/ .............. Unix style man-pages
+      |-- plugins/ .......... Documentation for our core plugins
+      |-- sdk/ .............. Plugin developer guide
+  |-- example/ .............. Example plugins
+  |-- iocore/ ...............
+      |-- aio/ .............. Asynchronous I/O core
+      |-- cache/ ............ Disk and RAM cache
+      |-- eventsystem/ ...... Event Driven Engine
+      |-- dns/ .............. DNS (asynchronous)
+      |-- hostdb/ ........... Internal DNS cache
+      |-- net/ .............. Network
+      |-- utils/ ............ Utilities
+  |-- lib/ ..................
+      |-- cppapi/ ........... C++ api wrapper for plugin developers
+      |-- perl/ ............. Perl libraries for e.g. mgmt access and configurations
+      |-- records/ .......... Library for config files
+      |-- ts/ ............... Base / core library
+      |-- tsconfig/.......... New config parser and library (experimental)
+      |-- wccp/ ............. WCCP implementation
+      |-- yamlcpp/ .......... Library for YAML of C++
+  |-- mgmt/ ................. Management server and tools
+  |-- plugins/ .............. Stable core plugins
+      |-- experimental/ ..... Experimental core plugins
+  |-- proxy/ ................ HTTP proxy logic
+      |-- api/ .............. Plugin API includes
+      |-- config/ ........... Configurations
+      |-- hdrs/ ............. Headers parsing and management
+      |-- http/ ............. The actual HTTP protocol implementation
+      |---http2/ ............ HTTP/2 implementation
+      |-- logging/ .......... Flexible logging
+      |-- shared/ ........... Shared files
+  |-- rc/ ................... Installation programs and scripts
+  |-- src/ .................. Source for all the main binaries / applications
+      |-- traffic_cache_tool/ Tool to interact with the Traffic Server cache
+      |-- traffic_crashlog/ . Helper process that catches Traffic Server crashes
+      |-- traffic_ctl/ ...... Command line manipulate and management tool
```

Comment:
I'd just say "Command line management tool". Or at least change manipulate to manipulation.